docs/index.md

```diff
@@ -15,11 +15,11 @@ metadata access for data and AI assets.
 
 ## Downloading
 
-You can get Graviton from the [GitHub release page](https://github.com/datastrato/gravitino/releases),
-or you can build Gravitino from source, please see [How to build Gravitino](./how-to-build.md).
+You can get Gravitino from the [GitHub release page](https://github.com/datastrato/gravitino/releases),
+or you can build Gravitino from source code, please see [How to build Gravitino](./how-to-build.md).
 
-Gravitino runs on both Linux and macOS, and requires Java 8. Gravitino trino-connector runs with
-Trino, and requires Java 17. This should include JVMs on x86_64 and
+Gravitino runs on both Linux and macOS, and requires at least Java 8. Gravitino trino-connector runs with
+Trino, and requires at least Java 17. This should include JVMs on x86_64 and
```

Comment:
@jerryshao 
Please take a look at this point.  I think we need to distinguish between the compilation environment and the running environment.

Reply:
Here, it means that trino requires Java 17 runtime, so our trino-connector is compiled against JDK17.

Reply:
Also, Gravitino currently can only run on Java 8, not on Java 11 or 17, so your change of "at least" is not correct.

Reply:
Besides, here it mentions all about the running environment, not the building environment.

Reply:
So the word `run` means `compilation` and `run`?

As far as I see, we can run the Gravitino server with JDK 17.

Reply:
I mean we need JDK 8 to 17 to build the Gravitino server and at least JRE 8 to run Gravitino server and require at least JRE 17 to run the Trino connector.

Reply:
It just means run, and it cannot run with JDK17 currently because of JAXB problem. You can try it.

Reply:
It works well for me after https://github.com/datastrato/gravitino/pull/868
![Uploading image.png…]()

Reply:
I think we don't have a thorough test, we should not guarantee this and modify this document to say that.

Reply:
My previous test has many issues related to jaxb.

Reply:
Okay, I will revert changes here.